pcip/README.md

```diff
@@ -0,0 +1,63 @@
+# Pulsar Java Contrib Improvement Proposal (PCIP)
```

Comment:
This is very close to the PIP process.
Do we have a discussion on the dev ML ?

Reply:
mail disscuss link: https://lists.apache.org/thread/z4bx7207cmprvp0vj8mgf1jhs7q2s4vn